test/gracenote/response_test.rb

```diff
@@ -2,23 +2,6 @@
 
 module Gracenote
   describe "client" do
-    describe "#register" do
-      it "should return the userid" do
-        stub_request(:post, //).to_return(headers: { 'Content-Type' => 'applicaiton/xml' }, body: <<-XML
-<RESPONSES>
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)